src/attached_probe.cpp

```diff
@@ -401,13 +405,17 @@ static void check_alignment(std::string &path,
     case AlignState::Ok:
       return;
     case AlignState::NotAlign:
-      if (safe_mode)
-        throw FatalUserException("Could not add " + probetypeName(type) +
-                                 " into middle of instruction: " + tmp +
-                                 std::string{ hint_unsafe });
-      else
+      if (safe_mode) {
+        auto msg = "Could not add " + probetypeName(type) +
+                   " into middle of instruction: " + tmp +
+                   std::string{ hint_unsafe };
+        if (orig_name.find('*') != std::string::npos)
```

Comment:
Should we also add this hint to the other failing case statements below?

Reply:
@ttreyer I meant using `hint_symbol_source` for the AlignState::Fail and AlignState::NotSupp - or do those not apply?

Reply:
I understand `AlignState::Fail` and `AlignState::NotSupp` to be errors returned by the disassembler, not necessary an issue with the address itself. That's why I'm not convinced that the hint applies to both of them.

Reply:
Would these failure occur if the user switched to `symbol_table` mode?

Reply:
Yes, the failure would still occur.

Reply:
Ok cool. I'm satisfied then 😄